spec/javascripts/section/section_page_spec.jsx

```diff
@@ -80,4 +81,28 @@ describe('SectionPage', function() {
       expect(firstDataRows[0].innerHTML).toEqual('<a href="/students/2">Duck, Donald</a>');
     });
   });
+
+  it('renders correctly', () => {
+    const props = {
```

Comment:
One pattern I like in test files is:

```
function testProps(props) {
  return {
    ...props,
     foo: 'default'
  };
}
...
const props = testProps({ bar: 'cfdsya' });
````

Reply:
(as a way to modify default fixture values so you don't have to duplicate them). 
 There's also a `fixtures` folder or file for common ones (like a default `currentEducator`).